packages/next/src/client/components/errors/stitched-error.ts

```diff
@@ -1,48 +1,33 @@
 import React from 'react'
 import isError from '../../../lib/is-error'
-import { copyNextErrorCode } from '../../../lib/error-telemetry-utils'
 
-const REACT_ERROR_STACK_BOTTOM_FRAME = 'react-stack-bottom-frame'
-const REACT_ERROR_STACK_BOTTOM_FRAME_REGEX = new RegExp(
-  `(at ${REACT_ERROR_STACK_BOTTOM_FRAME} )|(${REACT_ERROR_STACK_BOTTOM_FRAME}\\@)`
-)
```

Comment:
We no longer stitch stacks together in the getter but once we add these errors to the store. We can delay that even further up until we actually look at the error if we want to display Owner Stacks differently. Right now it's not obvious what the sync Stack and what the Owner Stack is.